Extension/bin/msvc.64.darwin.json

```diff
@@ -2,8 +2,7 @@
     "defaults": [
       "--clang",
       "--pack_alignment",
-      "8",
-      "--framework_include_directory=/System/Library/Frameworks"
+      "8"
```

Comment:
Is this a required change?

Reply:
This should go in at some point though it's not required to be part of this change. This repo is out of sync with the language server.